packages/app-accounts/src/util/accounts.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017-2018 @polkadot/app-accounts authors & contributors
+// This software may be modified and distributed under the terms
+// of the ISC license. See the LICENSE file for details.
+
+let accountsQty: (accountAll?: any[] | undefined) => number;
+let isAccounts: (accountAll?: any[] | undefined) => boolean;
+let isNoAccounts: (accountAll?: any[] | undefined) => boolean;
+let showIsOrAre: (accountAll?: any[] | undefined) => string;
+let showPlural: (accountAll?: any[] | undefined) => string;
+
+accountsQty = (accountAll?: any[] | undefined): number => isAccounts(accountAll) && accountAll ? Object.keys(accountAll).length : 0;
```

Comment:
Using `accountsAll?: any[]` implies `undefined` as well, i.e. the `?` does that, it says "this can be supplied or not"